### PR TITLE
fix(service-worker): ensure initialised in browser only

### PR DESCRIPTION
**Pull request by alxhub**

### CODE REVIEW

File: packages/service-worker/src/module.ts [hunk 1]
```diff
@@ -79,7 +82,11 @@ export class ServiceWorkerModule {
         {
           provide: APP_INITIALIZER,
           useFactory: ngswAppInitializer,
+<<<<<<< HEAD
           deps: [Injector, SCRIPT, RegistrationOptions],
+=======
+          deps: [Injector, SCRIPT, OPTS, PLATFORM_ID],
+>>>>>>> fix(service-worker): ensure initialised in browser only
```
Comment (gkalpak) [inline packages/service-worker/src/module.ts]: Ooops!

**Review by IgorMinar — DISMISSED** [review]
please explain the default value and resolve the merge conflict. otherwise lgtm. thanks!

File: packages/service-worker/src/low_level.ts [hunk 2]
```diff
@@ -86,8 +87,9 @@ export class NgswCommChannel {
    */
   readonly events: Observable<IncomingEvent>;
 
-  constructor(private serviceWorker: ServiceWorkerContainer|undefined) {
-    if (!serviceWorker) {
+  constructor(private serviceWorker: ServiceWorkerContainer|undefined,
+      @Inject(PLATFORM_ID) platformId: string = 'browser') {
```
[the remark this answers is not in the document]
Comment (alxhub) [reply packages/service-worker/src/low_level.ts]: Default value is only for tests - it'll be injected at runtime.

[the remark this answers is not in the document]
Comment (alxhub) [reply packages/service-worker/src/low_level.ts]: Removed.

**Review by IgorMinar — APPROVED** [review]
The CI is not happy - compile-time error. Please fix. Otherwise lgtm

**Comment by alxhub** [comment]
@IgorMinar should be fixed now.

**Pull request closed without merging**